### ISSUE TITLE
Implement the "share this page" button

### ISSUE THREAD

**Issue by nfee006** [asc-csa/Scisat-App#137]
Earlier attempts have been unsuccessful, likely due to interactions between the JS libraries used by wet-boew and by dash (see https://github.com/open-data/ckanext-canada/issues/1191).  

![image](https://user-images.githubusercontent.com/11013061/117986921-0e8a7f80-b308-11eb-987e-9ce8ea2b5a0e.png)

**Comment by nfee006** [comment]
4.    Add the «share this page » button for the two micro applications. NF note: I had done this but it hadn’t worked right. It needs to be added here: https://github.com/asc-csa/Scisat-App/commit/86551e35e7fcceb05288412c8ab64b068876a8f6. https://wet-boew.github.io/v4.0-ci/docs/ref/share/share-en.html
 
FYI @coleweusthem. Let me know if it's going to be an issue please.

**Comment by coleweusthem** [comment]
@nfee006, yeah it's not working correctly for me either. It's populating the button but for some reason clicking, it does nothing. It's not throwing any errors. I think either you're right and it's a conflict (Although I would expect to see some console errors), or there's a bug in wet-boew, or for some reason, it needs to be run on canada.ca. I'm not sure there's much I can do without so an indication of why it's not working. I'll leave this for last and take another look at it then, see if I can't figure it out

**Comment by emilinecsa** [comment]
This button works on [CSA' web page](https://www.asc-csa.gc.ca/eng/). I will talk to communications and fix this issue.
